File: Lista Aula 6/Ex10.py
### Exercício 10
'''
Reescreva o programa anterior lendo um número inteiro adicional chamado Prim. Nesta versão o
programa deverá apresentar os N termos da sequência de Fibonacci que forem maiores que Prim.
'''

num = int(input("Digite o número de termos: "))
while  num <= 0:
    num = int(input("Digite um número maior que zero: "))
    
Prim = int(input("Digite o valor de Prim: "))
cont = 0
n1 = 0
n2 = 1
n3 = 0

while num > cont:
    if n1 > Prim:
        print(n1)
        cont += 1
    n3 = n1 + n2
    n1 = n2
    n2 = n3

File: C/Ex10.c
#include <stdio.h>

int main(){
    //Atividade 04.2
    int num;
    
    printf("Digite um valor inteiro: ");
    scanf("%d", &num);
    while (num != 0){
        printf("%d\n", num * num);
        scanf("%d", &num);
    }

    return 0;
}